package.json

```diff
@@ -78,38 +62,6 @@
     "cozy-scripts": "0.4.1",
     "cozy-ui": "7.0.3",
     "preact": "8.2.7",
-    "preact-compat": "3.17.0",
-    "react-redux": "5.0.6",
-    "react-router-dom": "4.2.2",
```

Comment:
You don't need the react router neither?

Reply:
Not for now no, this app has a single view.

Reply:
As Kent Beck said in [« extreme programing explained »](https://pdfs.semanticscholar.org/a8e9/992fbe4b9c90121f8ff2bb22816d37dfebb2.pdf):
> You would only implement what you had to, in hopes that the needs you anticipate for tomorrow wouldn't come true.